packages/transformers/postcss/src/PostCSSTransformer.js

```diff
@@ -0,0 +1,195 @@
+// @flow
+
+import type {FilePath, MutableAsset} from '@parcel/types';
```

Comment:
Does this need to be in the dep list?

Reply:
Yarn workspaces links everything at the repo root so flow can find this fine. And since it's not a real runtime dependency (it's `import type`), this won't cause any issues when it comes to using published versions.

It certainly works for now, and I think if we decide one way or another, we should enforce it via lint. Right now I'm trying not to list any `@parcel` packages in `devDependencies` since it's not strictly necessary. Though as @padmaia mentioned this does couple the package to our current monorepo (specifically yarn workspace) structure and it would be ideal if packages can be trivially removed if necessary.

Reply:
I'm sure @jamiebuilds has thoughts on this 😃

Reply:
I am very on the fence with this, the wind could easily take me to either side. As for this I just wanted to ensure it was intentional.